recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -44,22 +55,25 @@ def requirements(self):
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
-
-        if tools.is_apple_os(self.settings.os):
-            # FIXME: add Apple support
-            raise ConanInvalidConfiguration("This recipe does not support Apple.")
-
-        if self.settings.os == "Linux":
```

Comment:
Seems odd this was removed

Reply:
Min version of gcc is set in _minimum_compilers_version.

Reply:
Ahhhhhh, I was referring to the message described why but that's also acceptable